src/v2/Apps/Order/Components/__tests__/ShippingQuotes.jest.tsx

```diff
@@ -65,6 +65,17 @@ describe("ShippingQuotes", () => {
     expect(shippingQuotes).toHaveLength(5)
   })
 
+  it("auto selects the first shipping quote", async () => {
+    const shippingQuotes = page
+      .find(`[data-test="shipping-quotes"]`)
+      .find(BorderedRadio)
+
+    // The id of the lowest cost shipping option in BuyOrderWithArtaShippingDetails mocked data
+    expect(shippingQuotes.first().props().value).toEqual(
+      "4a8f8080-23d3-4c0e-9811-7a41a9df6933"
+    )
+  })
```

Comment:
Hard to test and its not very clear the behavior here, a symptom of how the data is formatted when we get it from exchange + we do filtering by price value on the client. (lowest -> highest price)